config.yml

```diff
@@ -0,0 +1,11 @@
+server:
+  port: 8080
+
+log:
+  level: debug
+
+CSV:
+  FileName: /home/omar/Desktop/saved.csv
```

Comment:
Based on your project I will just add the template so anyone can fill up their own values.

Reply:
A recommendation here is to provide a `config.yml.dist` file instead of the actual `config.yml` file, and add instructions on what to do with this file.